src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -203,3 +205,34 @@ def run(self, args, unknown_args):
         args.shell_command = 'echo {{ inventory_hostname }}'
         args.silence_warnings = False
         return RunShellCommand(self.parser).run(args, unknown_args)
+
+
+class UpdateRiakSecrets(CommandBase):
+    command = 'update-riak'
```

Comment:
When I saw this above, the command name `update-riak` did not clearly communicate what aspect of riak was being updated. What do you think of renaming this to `update-riak-secrets`?

Reply:
Updated here https://github.com/dimagi/commcare-cloud/pull/2038/commits/5d32c8de9264f96efa4a76de1e0674fa19a3f6be